coretex/cli/commands/node.py

```diff
@@ -155,6 +155,7 @@ def config(verbose: bool) -> None:
 
 
 @click.group()
+@onBeforeCommandExecute(node_module.checkResourceLimitations)
```

Comment:
If the `checkResourceLimitations` command gets executed on every command from the `node` group doesn't that mean it will get executed when I call `coretex node config`, and if by any chance I have invalid configuration it will fail?

Reply:
This should call the same function for validating node configuration, and there should be only one function which does that to keep it straightforward, bug-prone, and easy to modify.

Reply:
My idea was to prevent users that dont have enough resources that are mandatory (for example RAM) from using the node functionality since they will not be able to do anything.